scriptmodules/system.sh

```diff
@@ -388,18 +388,17 @@ function get_platform() {
                 __platform="armv7-mali"
                 ;;
             *)
-                # jetson nano and tegra x1 can be identified via /sys/firmware/devicetree/base/model
-                local model_path="/sys/firmware/devicetree/base/model"
-                if [[ -f "$model_path" ]]; then
-                    # ignore end null to avoid bash warning
-                    local model=$(tr -d '\0' <$model_path)
-                    case "$model" in
-                        "NVIDIA Jetson Nano Developer Kit")
-                            __platform="jetson-nano"
-                            ;;
-                        icosa|icosa_emmc)
-                            __platform="tegra-x1"
-                            ;;
+                # jetsons can be identified by their dtsfilename
+                # see https://github.com/rbonghi/jetson_stats/blob/master/jtop/jetson_variables for similar implementation
+                if [ -f /proc/device-tree/nvidia,dtsfilename ]; then
+                    local JETSON_CODENAME=$(tr -d '\0' < /proc/device-tree/nvidia,dtsfilename)
```

Comment:
Please can you use lowercase variable names

Reply:
I'm going to make all of these as resolved since I've updated the files